oap-ape/README.md

```diff
@@ -72,7 +72,7 @@ mvn clean install
 Native and Java build. We will use `$OAP_ROOT_DIR/oap-ape/ape-java/ape-spark/target/ape-spark-1.1.0-SNAPSHOT.jar`, `$OAP_ROOT_DIR/oap-ape/ape-java/ape-flink/ape-flink-x.xx.x/target/ape-flink-x.xx.x-1.1.0-SNAPSHOT.jar` later.
 ```
 cd $OAP_ROOT_DIR/oap-ape/ape-java
-mvn clean package -am -Pshading
+mvn clean package -am --batch-mode -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
```

Comment:
I think we should not change `-Pshading`, any comments @ZJie1

Reply:
yes, we should not change it. This is for the packing native to the ape spark jar and ape-server also use this.

Reply:
But I cannot compile successfully if I use the -Pshading command. 
Here is the error

[ERROR] Failed to execute goal org.apache.maven.plugins:maven-shade-plugin:3.2.4:shade (default) on project ape-server: Error creating shaded jar: Could not resolve following dependencies: [com.intel.oap:ape-flink-common:jar:1.1.0-SNAPSHOT (compile), com.intel.oap:ape-client:jar:1.1.0-SNAPSHOT (compile), com.intel.oap:ape-hcfs:jar:1.1.0-SNAPSHOT (compile)]: Could not resolve dependencies for project com.intel.oap:ape-flink-1.13.2:jar:1.1.0-SNAPSHOT: The following artifacts could not be resolved: com.intel.oap:ape-flink-common:jar:1.1.0-SNAPSHOT, com.intel.oap:ape-client:jar:1.1.0-SNAPSHOT, com.intel.oap:ape-hcfs:jar:1.1.0-SNAPSHOT: Could not find artifact com.intel.oap:ape-flink-common:jar:1.1.0-SNAPSHOT -> [Help 1]